src/amr/data/tensorfield/tensor_field_geometry.hpp

```diff
@@ -0,0 +1,173 @@
+#ifndef PHARE_SRC_AMR_TENSORFIELD_TENSORFIELD_GEOMETRY_HPP
+#define PHARE_SRC_AMR_TENSORFIELD_TENSORFIELD_GEOMETRY_HPP
+
+
+#include "amr/data/field/field_geometry.hpp"
+#include "amr/data/tensorfield/tensor_field_overlap.hpp"
+#include "core/def/phare_mpi.hpp" // IWYU pragma: keep
```

Comment:
this is a special header and needs to be before any samrai (or /amr) includes

it's got some `#define`s so that we silence somethings during compilation